.chloggen/servicegraph-fix-time-unit.yaml

```diff
@@ -0,0 +1,31 @@
+# Use this changelog template to create an entry for release notes.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: bug_fix
```

Comment:
Even though this is a breaking change, I think the change type is correctly set to bug fix.